test/k8sT/Services.go

```diff
@@ -2760,6 +2760,42 @@ Secondary Interface %s :: IPv4: (%s, %s), IPv6: (%s, %s)`, helpers.DualStackSupp
 					testNodePort(true, false, false, 0) // no need to test from outside, as testDSR did it
 				})
 
+				SkipContextIf(helpers.DoesNotRunWithKubeProxyReplacement, "BPF NAT engine handles unknown protocol packets", func() {
+					var (
+						iperf3Manifest string
+						podsIps        map[string]string
+						iperf3Server   = "iperf3-server"
+					)
+					BeforeAll(func() {
+						// DeployCiliumAndDNS(kubectl, ciliumFilename)
```

Comment:
Probably best to redeploy Cilium with the default configuration given it was deployed with non-defaults on line 2753.